src/todos/todos.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@nestjs/common';
+
+import { CreateTodoDto } from './create-todo.dto';
+import { Todo } from 'src/common/entities/todo.entity';
+import { TodoRepository } from './todos.repository';
+
+@Injectable()
+export class TodosService {
+  constructor(private todoRepository: TodoRepository) {}
+
+  async createTodos(createTodoDto: CreateTodoDto): Promise<Todo> {
```

Comment:
```suggestion
  async create(createOne: CreateOneDto): Promise<Todo> {
```

This is in the todosService so naming it createOne should be fine.